msg/BoundingBox.msg

```diff
@@ -1,7 +1,7 @@
 # Note: This message plans to move to geometry_msgs in the final implementation.
 
-# position and rotation of the bounding box
+# position and orientation of the bounding box center
 geometry_msgs/Pose pose
 
 # size of the bounding box
-geometry_msgs/Vector3 size
\ No newline at end of file
+geometry_msgs/Point size
```

Comment:
Why change this to a Point?

This should be documented how to resolve this. What is it relative to? The pose of the `pose`?